src/main/java/seedu/address/logic/Logic.java

```diff
@@ -24,4 +25,6 @@
 
     /** Returns the list of input entered by the user, encapsulated in a {@code ListElementPointer} object */
     ListElementPointer getHistorySnapshot();
+
+    ObservableList<Cca> getFilteredCcaList();
```

Comment:
Might want to add a comment to explain what this does?

Reply:
Added comments.